src/sentry/quotas/base.py

```diff
@@ -657,11 +693,24 @@ def check_accept_monitor_checkin(self, project_id: int, monitor_slug: str):
 
         return PermitCheckInStatus.ACCEPT
 
+    def check_accept_checkin(self, data_category: DataCategory, project_id: int, slug: str):
+        """
+        Will return a `PermitCheckInStatus`.
+        """
+        from sentry.monitors.constants import PermitCheckInStatus
+
+        return PermitCheckInStatus.ACCEPT
+
     def update_monitor_slug(self, previous_slug: str, new_slug: str, project_id: int):
         """
         Updates a monitor seat assignment's slug.
         """
 
+    def update_seat_slug(self, previous_slug: str, new_slug: str, project_id: int):
```

Comment:
In the case of uptime monitors there are no slugs.

I do think this interface probably makes sense in the general sense however.

Reply:
the slug for uptime will just be the UUID assigned to the monitor.

Reply:
The the UUID a primary key?
Can it be updated in future?

Reply:
no the UUID will never change on a uptime monitor.

Reply:
Wouldn't it just be the id of the row in the db?

Reply:
You’re right I think I’m mixing the up with crons